scipy/sparse/tests/test_base.py

```diff
@@ -658,6 +658,12 @@ def test_abs(self):
         A = matrix([[-1, 0, 17],[0, -5, 0],[1, -4, 0],[0,0,0]],'d')
         assert_equal(abs(A),abs(self.spmatrix(A)).todense())
 
+    def test_round(self):
+        decimal = 1
+        A = matrix([[-1.35, 0.56],[17.25, -5.98]],'d')
```

Comment:
PEP 8: spaces after the commas.  (I see that this happens elsewhere in the file, but we should not make it worse with new code. :)